datafusion/sql/src/unparser/plan.rs

```diff
@@ -502,3 +505,35 @@ impl From<BuilderError> for DataFusionError {
         DataFusionError::External(Box::new(e))
     }
 }
+
+#[cfg(test)]
+mod test {
+    use crate::unparser::plan_to_sql;
+    use arrow::datatypes::{DataType, Field, Schema};
+    use datafusion_expr::{col, logical_plan::table_scan};
+    #[test]
+    fn test_plan_to_sql() {
```

Comment:
```suggestion
    fn test_table_references_in_plan_to_sql() {
```

This test technically just covers the table references part.